std/zig/tokenizer.zig

```diff
@@ -235,6 +236,7 @@ pub const Tokenizer = struct {
         StringLiteralBackslash,
         MultilineStringLiteralLine,
         CharLiteral,
+        CharLiteralUTF8,
```

Comment:
is this state ever entered? (is this code unfinished?)

Reply:
Yes.